surface/include/pcl/surface/3rdparty/poisson4/sparse_matrix.hpp

```diff
@@ -250,22 +250,22 @@ namespace pcl
     template<class T>
     SparseMatrix<T>& SparseMatrix<T>::operator *= (const T& V)
     {
-      for (int i=0; i<this->Rows(); i++)
+      for (int i=0; i<rows; i++)
       {
-        for(int ii=0;ii<m_ppElements[i].size();i++){m_ppElements[i][ii].Value*=V;}
+        for(int ii=0;ii<rowSizes[i];i++){m_ppElements[i][ii].Value*=V;}
```

Comment:
Would you also fix `i++` to `ii++` here?

Reply:
Sorry - this got buried in my inbox.

Reply:
No problem, I took care of it :smile: